#Expyriment 1.0.0 (Python 3.13.7), .xpe-file, coding: cp1252
#date: Thu Sep 25 2025 10:29:07
#sha1: 4f5941
#modules: 
#display: size=(800, 600), window_mode=True, opengl=2
#os: uname_result(system='Windows', node='zlr778', release='11', version='10.0.22631', machine='AMD64')
Time,Type,Event,Value,Detail,Detail2
1919,design,log,
#design: #exp: kanizsa-square
#design: block_cnt,block_id,trial_cnt,trial_id
1919,design,logged,
1919,Experiment,started
1921,stderr,received,'Traceback (most recent call last):'
1922,stderr,received,'  File "C:\\Users\\20946\\CogSup\\Programming\\Assignments\\Week-3\\Exercises\\kanizsa-square.py", line 32, in <module>\n    circle = stimuli.Circle(radius = stim_length, colour = colour, position = i)\n                                                                              ^'
1922,stderr,received,"NameError: name 'i' is not defined. Did you mean: 'id'?"
